src/frontend/src/btc/schedulers/wallet.scheduler.ts

```diff
@@ -0,0 +1,58 @@
+import { getBtcBalance } from '$lib/api/signer.api';
+import { WALLET_TIMER_INTERVAL_MILLIS } from '$lib/constants/app.constants';
+import { SchedulerTimer, type Scheduler, type SchedulerJobData } from '$lib/schedulers/scheduler';
+import type {
+	PostMessageDataRequestBtc,
+	PostMessageDataResponseBtcWallet
+} from '$lib/types/post-message';
+import { assertNonNullish } from '@dfinity/utils';
+
+export class WalletScheduler implements Scheduler<PostMessageDataRequestBtc> {
```

Comment:
`btc-wallet.scheduler.ts` and `BtcWalletScheduler` maybe? Can be handy when we lookup for the module

Reply:
https://github.com/dfinity/oisy-wallet/pull/2498